doc/10-icinga-template-library.md

```diff
@@ -2519,6 +2521,16 @@ icingacli_businessprocess_config          | **Optional.** Configuration file con
 icingacli_businessprocess_details         | **Optional.** Get details for root cause analysis. Defaults to false.
 icingacli_businessprocess_statetype       | **Optional.** Define which state type to look at, `soft` or `hard`. Overrides the default value inside the businessprocess module, if configured.
 
+#### Director <a id="plugin-contrib-icingacli-director"></a>
+
+This subcommand is provided by the [director module](https://github.com/Icinga/icingaweb2-module-director) > 1.4.2 and executed as `icingacli director health check`. Please refer to the [documentation](https://github.com/Icinga/icingaweb2-module-director/blob/master/doc/60-CLI.md#health-check-plugin) for all available sub-checks.
+
+Custom attributes passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                                      | Description
+------------------------------------------|-----------------------------------------------------------------------------------------
+icingacli_director_check                  | **Optional** Run only a specific test suite
+icingacli_director_db                     | **Optional** Use a specific Icinga Web DB resource
```

Comment:
Just a very little cosmic thing: The punctuation marks are missing :)

Reply:
I was quite sure I double-checked before committing. Meh. Thanks for finding the missing marks.